pkg/scheduler/framework/plugins/nodevolumelimits/non_csi_test.go

```diff
@@ -36,11 +37,6 @@ import (
 )
 
 var (
-	oneVolPod = st.MakePod().Volume(v1.Volume{
-		VolumeSource: v1.VolumeSource{
-			AWSElasticBlockStore: &v1.AWSElasticBlockStoreVolumeSource{VolumeID: "ovp"},
-		},
-	}).Obj()
 	twoVolPod = st.MakePod().Volume(v1.Volume{
```

Comment:
Sorry, I should be more precise, I mean all these pods only used in `TestEBSLimits`, so we should move all of them to this function as local variables. including `twoVolPod`, `splitVolsPod`, `nonApplicablePod`...

Reply:
Ok, `nonApplicablePod` also used by `TestAzureDiskLimits`.